lxd/logging.go

```diff
@@ -108,12 +104,12 @@ func expireLogs(ctx context.Context, state *state.State) error {
 			for _, logfile := range logs {
 				path := shared.LogPath(entry.Name(), logfile.Name())
 
-				// Always keep the LXC config
-				if logfile.Name() == "lxc.conf" {
+				// Always keep the config files.
+				if logfile.Name() == "lxc.conf" || logfile.Name() == "qemu.conf" {
```

Comment:
@stgraber are there any files that end with ".conf" that we don't want to keep? If not then could this just be `strings.HasSuffix(logfile.Name(),".conf")` to avoid having to duplicate the knowledge of the name of each instance type's conf file name here?